src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
```

Comment:
Avoid using magic numbers like '1vmin' for the border. Instead, consider defining a variable for such values to make the code more maintainable and understandable. For example: `$border-width: 1vmin;` and then `border: $border-width dotted #000;`.